include/onnxruntime/core/session/onnxruntime_run_options_config_keys.h

```diff
@@ -42,3 +42,9 @@ static const char* const kOrtRunOptionsConfigQnnPerfModePostRun = "qnn.htp_perf_
 
 // Set RPC control latency for QNN HTP backend
 static const char* const kOrtRunOptionsConfigQnnRpcControlLatency = "qnn.rpc_control_latency";
+
+// Set graph annotation id for CUDA EP. Use with enable_cuda_graph=true.
+// The value should be an integer. If the value is not set, ORT session only captures one cuda graph.
```

Comment:
`ORT session only captures one cuda graph` is not true when user calls twice, once the id is not set, another time the id is set to 1. We will capture two graphs.

How about change the description to `If the value is not set, the default value is 0.`.

Reply:
I think the original meaning is "If the value is not set in the lifetime of ort session" so it is expected that only one graph is captured